styles.css

```diff
@@ -0,0 +1,45 @@
+body {
```

Comment:
It would be better to fix linter errors like:
- Expected indentation of 2 spaces